### PR TITLE
chore(dockerfile): change deprecated maintainer instructor to label

### PR DESCRIPTION
**Pull request by KeisukeYamashita**
## What

As title. The `MAINTAINER` label is deprecated and now we should use `LABEL` instead.

## Ref

https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fieldju) [inline .idea/gradle.xml:1]: Why did you delete this file, that doesn't seem like it should be part of this PR.

I do believe that @plumpy added all the IDE files explicitly.

Comment (KeisukeYamashita) [reply .idea/gradle.xml]: @fieldju Sorry, I've deleted this by mistake. Recovered it 🙇
